Source/SwiftLintFramework/Rules/MarkRule.swift

```diff
@@ -24,11 +24,14 @@ public struct MarkRule: CorrectableRule, ConfigurationProviderRule {
     public static let description = RuleDescription(
         identifier: "mark",
         name: "Mark",
-        description: "MARK comment should be in valid format.",
+        description: "MARK comment should be in valid format. e.g. // MARK: ok",
```

Comment:
Happy to add it the extended help. (I personally love how the `-` becomes a line in the Xcode navigator.)